src/Compilers/VisualBasic/Portable/Lowering/StateMachineRewriter/StateMachineStates.vb

```diff
@@ -8,9 +8,9 @@ Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
 Namespace Microsoft.CodeAnalysis.VisualBasic
 
     Friend Module StateMachineStates
-        Public FinishedStateMachine As Integer = -2
-        Public NotStartedStateMachine As Integer = -1
-        Public FirstUnusedState As Integer = 0
+        Public Const FinishedStateMachine As Integer = -2
```

Comment:
This is (at first glance) a good change. Could you submit just this change in a PR?